src/applications/disability-benefits/all-claims/tests/pages/vaMedicalRecords.unit.spec.jsx

```diff
@@ -63,7 +63,8 @@ describe('VA Medical Records', () => {
     );
 
     form.find('form').simulate('submit');
-    expect(form.find('.usa-input-error-message').length).to.equal(6);
+    // Required fields: Facility name, related disability, and treatment start date
+    expect(form.find('.usa-input-error-message').length).to.equal(3);
```

Comment:
Hmm, `country` is also required, but set to `USA` by default, right?

Reply:
Yup. Exactly it.